.github/workflows/test.yaml

```diff
@@ -0,0 +1,34 @@
+name: Run tests
+
+on:
+  pull_request_target:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        # This is the version of the action for setting up Python, not the Python version.
+        uses: actions/setup-python@v5
+        with:
+          # Semantic version range syntax or exact version of a Python version
+          python-version: '3.x'
+      - name: Display Python version
+        run: python -c "import sys; print(sys.version)"
+      - name: Install poetry
+        # see https://github.com/marketplace/actions/setup-poetry
+        uses: Gr1N/setup-poetry@v9
+        with:
+          poetry-version: 1.8.1
+      - name: Install dependencies
+        run: poetry install
+      - name: Run tests
+        run: |
+          python -m unittest github_standards/test_standards.py
```

Comment:
We should wrap this in `poetry` too so the correct Virtual Environment is used - so `poetry run python -m unittest ...`

Reply:
Aha! So that's why my local CI build kept failing. Thanks! Fixed in [43635f3](https://github.com/sonatype-nexus-community/github-management/pull/4/commits/43635f3b533299fcb4b3c4454471fc55c42f6c8f)
Also needed latest poetry version, I think...though not 100% certain...there was a bit of flailing going on.